### ISSUE TITLE
www: adjust rate limits

### ISSUE THREAD

**Issue by peterthomassen** [desec-io/desec-stack#304]
@nils-wisiol The issue with https://dedyn.io/check that was observed yesterday is actually not related to CORS headers, but to nginx rate limiting. We allow 30r/s on /api/ (which is used by the check script). Although that should suffice, rate limiting information was logged w.r.t. these requests.

I went on to reproduce rate limiting kicking in, below the configured rate. Just check this out:

    $ for i in `seq 1 5`; do curl -s -I https://desec.io/api/v1/ & done | grep HTTP
    HTTP/2 429 
    HTTP/2 200 
    HTTP/2 429 
    HTTP/2 429 
    HTTP/2 429 

When I digged depper, I found [this](https://www.nginx.com/blog/rate-limiting-nginx/#Configuring-Basic-Rate-Limiting):

> Rate – Sets the maximum request rate. In the example, the rate cannot exceed 10 requests per second. NGINX actually tracks requests at millisecond granularity, so this limit corresponds to 1 request every 100 milliseconds. Because we are not allowing for bursts (see the next section), this means that a request is rejected if it arrives less than 100 milliseconds after the previous permitted one.

In other words, with our rate, throttling will kick in if two requests arrive less than 33ms apart. It's a bit unexpected that [this is not mentioned in the nginx documentation itself](https://nginx.org/en/docs/http/ngx_http_limit_req_module.html).

[The appropriate way to handle this](https://www.nginx.com/blog/rate-limiting-nginx/#bursts) it setting the burst to 29.
